File: 2021-01-10-token-enhancer/action-in-blog-resource/src/main/java/blog/in/action/ActionInBlogApplication.java
package blog.in.action;

import blog.in.action.entity.Member;
import blog.in.action.repository.MemberRepository;
import lombok.RequiredArgsConstructor;
import org.springframework.boot.CommandLineRunner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@RequiredArgsConstructor
@SpringBootApplication
public class ActionInBlogApplication implements CommandLineRunner {

    private final MemberRepository memberRepository;

    public static void main(String[] args) {
        SpringApplication.run(ActionInBlogApplication.class, args);
    }

    @Override
    public void run(String... args) throws Exception {
        memberRepository.save(Member.builder()
                .id("Junhyunny")
                .name("Junhyunny")
                .email("junhyunny@naver.com")
                .address("Seoul")
                .build()
        );
    }
}

File: 2024-02-07-timezone-problem-of-jvm-application/action-in-blog/src/main/kotlin/blog/in/action/ActionInBlogApplication.kt
package blog.`in`.action

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class ActionInBlogApplication

fun main(args: Array<String>) {
    // System.setProperty("user.timezone", "Asia/Seoul")
    runApplication<ActionInBlogApplication>(*args)
}
